Dockerfile

```diff
@@ -1,12 +1,12 @@
 FROM jenkins
+
 USER root
+RUN apt-get -qq update \
+   && apt-get -qq -y install \
+   curl
+
+RUN curl -sSL https://get.docker.com/ | sh
 
```

Comment:
The reason why I do a manual download is to only keep the docker client, not the whole docker distribution. This saves some disk space. If you keep my original lines for those, I'll accept your PR